File: ruby/code_wars/keep_up_the_hoop.rb
# The `hoop_count` method takes an integer `n` as an argument.
# This method is used to determine the level of a hula hoop player.
#
# @param n [Integer] The number of hoops a player can hoop.
# @return [String] A motivational message based on the player's skill level.
def hoop_count n
  # If the player can hoop 10 or more hoops, they are considered skilled and are encouraged to learn tricks.
  # Otherwise, they are encouraged to keep practicing.
  n >= 10 ? "Great, now move on to tricks" : "Keep at it until you get it"
end

File: python/code_wars/keep_up_the_hoop.py
def hoop_count(n):
    """
    This function determines the feedback for a hula hoop performer based on the number of hoops they can handle.

    Parameters:
    n (int): The number of hoops the performer can handle.

    Returns:
    str: Encouraging message for the performer. If the performer can handle 10 or more hoops, they are encouraged to move on to tricks.
         Otherwise, they are encouraged to keep practicing.
    """
    return "Great, now move on to tricks" if n >= 10 else "Keep at it until you get it"
